sherpa/models/model.py

```diff
@@ -46,12 +46,12 @@ def cache_model(cls, pars, xlo, *args, **kwargs):
         digest = ''
         if use_caching:
 
-            data = [ numpy.array(pars).tostring(), str(kwargs.get('integrate',0)), 
+            data = [ numpy.array(pars).tostring(), kwargs.get('integrate', b'0'),
```

Comment:
So, if `integrate` is defined in `kwargs`, it is likely to be True or False. Is that not going to cause a problem in line 54 below - e.g. in python 2.7 I get

```
>>> a = ["foo", True, "bar"]
>>> b''.join(a)
TypeError: sequence item 1: expected string, bool found
```

Or have we guaranteed that by the time we call this routine we are guaranteed that the integrate keyword is a string?